specification/vi/resource-manager/readme.go.md

```diff
@@ -2,12 +2,21 @@
 
 These settings apply only when `--go` is specified on the command line.
 
-```yaml $(go)
+```yaml $(go) && !$(track2)
 go:
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: vi
   clear-output-folder: true
 ```
+
+``` yaml $(go) && $(track2)
+license-header: MICROSOFT_MIT_NO_VERSION
+module-name: sdk/resourcemanager/vi/armvi
```

Comment:
well, I do not like the name `vi`... could we use `videoindexer` instead to get a more descriptive name?